push_notifications.go

```diff
@@ -8,20 +8,33 @@ import (
 	"net/http"
 	"regexp"
 	"time"
+	"unicode/utf8"
 
+	"github.com/dgrijalva/jwt-go"
 	"github.com/pkg/errors"
 )
 
 // The Pusher Push Notifications Server API client
 type PushNotifications interface {
 	// Publishes notifications to all devices subscribed to at least 1 of the interests given
-	// Returns a non-empty `publishId` string if successful; or a non-nil `error` otherwise.
+	// Returns a non-empty `publishId` JSON string if successful; or a non-nil `error` otherwise.
+	PublishToInterests(interests []string, request map[string]interface{}) (publishId string, err error)
+	// An alias for `PublishToInterests`
```

Comment:
```
Deprecated: Use `PublishToInterests` instead.
```

Following this: https://rakyll.org/deprecated/. GoLand supports these deprecated comments, for example.